src/app/zap-templates/common/ClusterTestGeneration.js

```diff
@@ -826,6 +826,28 @@ function isLiteralNull(value, options)
   return (value === null) || (value instanceof NullObject);
 }
 
+function isHexString(value)
+{
+  return value && value.startsWith('hex:');
+}
+
+function octetStringFromHexString(value)
+{
+  const hexString = value.substring(4);
+
+  const bytes = [];
+  for (let i = 0; i < hexString.length; i += 2) {
+    bytes.push(hexString.substring(i, i + 2));
+  }
```

Comment:
Not sure whether this is clearer than:
```suggestion
  const bytes = hexString.match(/(..)/g);
```
or not... But also, can/should we throw an error if the number of chars is odd or some char is not in `[0-9a-fA-F]`?

Reply:
I have updated the code with your suggestion and I have also added errors if the number of characters is odd and if there is any invalid hexadecimal characters.